.circleci/config.yml

```diff
@@ -1,6 +1,40 @@
 version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
+  win: circleci/windows@4.1
+
+jobs:
+  windows_test:
+    executor: 
+      name: win/default
+      size: medium
+      shell: cmd.exe
+
+    steps: 
+      - checkout
+
+      - restore_cache:
+          keys:
+            [choco-cache]
+          
+      - run:
+          name: "Build"
+          command: choco install -y erlang elixir
```

Comment:
That's not building, maybe something like `Install elixir` would be a better name